app/views/News.js

```diff
@@ -70,11 +70,7 @@ class NewsScreen extends Component {
             borderBottomRightRadius: 12,
           }}
           cacheEnabled
-          onLoad={() =>
-            this.setState({
-              visible: false,
-            })
-          }
+          startInLoadingState={true} //Show loading indicator on web view
```

Comment:
So @eyganeshv - your IDE should be showing you this error

<img width="681" alt="Screen Shot 2020-05-11 at 8 51 03 PM" src="https://user-images.githubusercontent.com/20157849/81629630-324f7880-93c9-11ea-926a-8941b1f0eb4e.png">

What that means is that since the value being passed to that prop is just `true`, you can omit it altogether.

```js
<WebView
          source={{
            uri: item.item.news_url,
          }}
          containerStyle={{
            borderBottomLeftRadius: 12,
            borderBottomRightRadius: 12,
          }}
          cacheEnabled
          startInLoadingState
        />
```

Reply:
Ok. fine